source/widgets/listview.c

```diff
@@ -577,6 +577,9 @@ void listview_set_num_elements(listview *lv, unsigned int rows) {
 }
 
 unsigned int listview_get_selected(listview *lv) {
+  if (lv->require_input && !lv->filtered) {
+    return UINT32_MAX;
+  }
```

Comment:
This is enough to avoid auto-selecting the first item when `require_input` is enabled and there's no input: